test/src/utils/error.spec.js

```diff
@@ -0,0 +1,17 @@
+import Error from '../../../src/utils/error/player-error'
+
+describe('Error class', () => {
+
+  let error = null;
+
+  beforeEach(() => {
+    error = new Error(Error.Severity.CRITICAL, Error.Category.MANIFEST, Error.Code.UNABLE_TO_GUESS_MANIFEST_TYPE, );
```

Comment:
`beforeEach` method should include generic operations for all the unit tests.
When u are creating specific error in it you missing the target of it.
Create in each unit the specific error and test it